Sectors/Static/LON/Worthing.txt

```diff
@@ -54,8 +54,8 @@ London AC Worthing          S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00
                             N051.39.11.000 W001.26.29.000 N051.37.49.000 W001.21.01.000
                             N051.37.49.000 W001.21.01.000 N051.35.26.000 W001.06.57.000
 
-                            N051.30.37.000 W000.38.30.000 N051.29.14.000 W000.28.00.000
-                            N051.29.14.000 W000.28.00.000 N051.17.35.000 W001.03.35.000
+                            N051.31.08.440 W000.40.37.890 N051.29.14.000 W000.28.00.000
+                            N051.31.08.440 W000.40.37.890 N051.17.35.000 W001.03.35.000
```

Comment:
It may well be, but can I just check is this right? They were different before but now both the same, and therefore line 58 doesn't follow from line 57 if that makes sense.